esp-radio/src/wifi/mod.rs

```diff
@@ -2689,6 +2694,30 @@ pub(crate) mod embassy {
     }
 }
 
+/// Power saving mode settings for the modem.
+#[non_exhaustive]
+#[derive(Default)]
+pub enum PowerSaveMode {
+    /// No power saving.
+    #[default]
+    None,
+    /// Minimum power save mode. In this mode, station wakes up to receive beacon every DTIM period.
+    Minimum,
+    /// Maximum power save mode. In this mode, interval to receive beacons is determined by the
+    /// `ESP_RADIO_CONFIG_LISTEN_INTERVAL` config option.
+    Maximum,
+}
+
+impl From<PowerSaveMode> for esp_wifi_sys::include::wifi_ps_type_t {
```

Comment:
This is code I think I wrote, but we need to change it 😅. This will make `esp_wifi_sys` part of the public API of esp-radio. See https://rust-lang.github.io/api-guidelines/necessities.html#public-dependencies-of-a-stable-crate-are-stable-c-stable